test/integration/targets/delegate_to/resolve_vars.yml

```diff
@@ -0,0 +1,15 @@
+---
+- hosts: localhost
```

Comment:
`name: though we test for 'vars' this is a backwards compatibility test and 'vars' variable will be deprecated and removed in the future.`

Reply:
done